resources/assets/templates/topic/topicCategoryForBreadcrumbs.hbs

```diff
@@ -0,0 +1 @@
+<a href="#categories/{{slug}}/topics">{{ name }}</a>
```

Comment:
Sorry, but seems like we don't need link here. Just text. But it should be like header text